### ISSUE TITLE
v0.0.0.65 Invalid port: 18333

### ISSUE THREAD

**Issue by walkjivefly** [Crowndev/crown#162]
Seen in a masternode debug.log
```
2022-10-30T17:45:49Z CActiveMasternode::ManageStatus() - Begin
2022-10-30T17:45:49Z CActiveMasternode::ManageStatus() - not capable: Invalid port: 18333 - 18333 is only supported on mainnet.
```

**Comment by walkjivefly** [comment]
This happened when a startmasternode was issued for a node which had stopped syncing and gone missing (at height 22320).

**Comment by BlockMechanic** [comment]
Fixed in 0.0.0.66

**Comment by walkjivefly** [comment]
Appears to be fixed in 0.0.0.69
